app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/Main.scala

```diff
@@ -28,6 +31,8 @@ import java.util.UUID
 
 object Server extends IOApp {
 
+  val timeout: FiniteDuration = new FiniteDuration(15, TimeUnit.SECONDS)
```

Comment:
We should probably make this config-able at some point -- see if, for example, killing requests after 5 seconds (or some other threshold) gets us anything over 15 seconds, but that's _much_ later optimization